src/main/java/apoc/util/FileUtils.java

```diff
@@ -124,20 +124,23 @@ public static boolean isFile(String fileName) {
     }
 
     public static PrintWriter getPrintWriter(String fileName, Writer out) throws IOException {
```

Comment:
what's the reason behind these changes?

Reply:
I couldn't get excel export workin correctly when acquiring a writer and converting it to a outputstream. With an output stream it works, so I changed this.